### DISCUSSION TITLE
Portal colors

### DISCUSSION THREAD

**Discussion by ItzAlmaZ** [BEEmod/BEE2-items#4130] [Feature Suggestions]
Make the color of the portals corresponding to the model for solo test chambers. (For p-body yellow and red, for atlas purple and blue)

**Comment by vrad-exe** [comment]
Not sure what you mean, are you referring to Coop autoportals?

**Reply by ItzAlmaZ** [reply]
Not auto-portals, but the colors themselves that portalgun puts, (the p-body/atlas model puts those portals that chell/bendy puts)

**Reply by ItzAlmaZ** [reply]
if possible

**Reply by vrad-exe** [reply]
Oh, that's not possible to implement

**Reply by SimpleFoxOfficial** [reply]
Autoportals in coop sounds good.

**Reply by FrozenDragon0** [reply]
> Autoportals in coop sounds good.

Those are already a thing in coop.

**Reply by vrad-exe** [reply]
They're not yet.

**Reply by FrozenDragon0** [reply]
> They're not yet.

Thought they were? Guess I must be mistaken then.
